assets/js/components/notifications/BannerNotification/index.js

```diff
@@ -201,14 +178,14 @@ function BannerNotification( props ) {
 			const event = new Event( 'notificationDismissed' );
 			document.dispatchEvent( event );
 		}, 350 );
-	}
+	};
 
 	const isNavigatingToCTALink = useSelect( ( select ) =>
 		ctaLink ? select( CORE_LOCATION ).isNavigatingTo( ctaLink ) : false
 	);
 
 	const { navigateTo } = useDispatch( CORE_LOCATION );
-	async function handleCTAClick( e ) {
+	const handleCTAClick = async ( e ) => {
```

Comment:
Please can you rename `e` here too?

```suggestion
	const handleCTAClick = async ( event ) => {
```